src/navigationStore.js

```diff
@@ -188,7 +188,11 @@ function createNavigationOptions(params) {
         || (init ? null : (renderBackButton && renderBackButton(state)) || <BackNavBarButton {...state} />);
     }
 
-    if (back) {
+    if (navigationParams.back != null) {
```

Comment:
please follow JS standards and use `!==` instead of `!=` as well as `!navigationParams.back` instead of `=== false`

Reply:
Right, thx. But first one was a typo by the way.